build/Versions.props

```diff
@@ -45,6 +45,7 @@
 
   <!-- Production Dependencies -->
   <PropertyGroup>
+    <LargeAddressAwareVersion>1.0.1</LargeAddressAwareVersion>
```

Comment:
What's the difference between "toolset" and "production"?

Reply:
I figured Repo Toolset was sort of a brand name?  Should I put this in that section?

Reply:
I have no idea; I was really asking :)